packages/target-electron/src/deltachat/stdio_server.ts

```diff
@@ -16,14 +17,52 @@ export class StdioServer {
     this.serverProcess = null
   }
 
-  async start() {
+  start() {
     this.serverProcess = spawn(this.cmd_path, {
       env: {
         DC_ACCOUNTS_PATH: this.accounts_path,
         RUST_LOG: process.env.RUST_LOG,
       },
     })
 
+    this.serverProcess.on('error', err => {
+      // The 'error' event is emitted whenever:
+      // - The process could not be spawned.
+      // - The process could not be killed.
+      // - Sending a message to the child process failed.
+      // - The child process was aborted via the signal option.
+      // ~ https://nodejs.org/api/child_process.html#event-error
+
+      if (err.message.endsWith('ENOENT')) {
+        dialog.showErrorBox(
+          'Fatal Error: Core Library Missing',
+          `The DeltaChat Module is missing! It should be located at "${
+            this.cmd_path
+          }".
+    
+This could be due to your antivirus program. Please check the quarantine to restore it and notify the developer about this issue.
+You can reach us on delta@merlinux.eu or on github.com/deltachat/deltachat-desktop/issues .
+
+The Log file is located in this folder: ${getLogsPath()}
+--------------------
+Error: ${err.message}
+`
+        )
+      } else {
+        dialog.showErrorBox(
+          'Fatal Error',
+          `Error with core has been detected, please contact developers: You can reach us on delta@merlinux.eu or on github.com/deltachat/deltachat-desktop/issues .
+
+          ${err.name}: ${err.message}
+
+          The Log file is located in this folder: ${getLogsPath()}\n
+          `
+        )
+      }
+      // I think we can exit in all the cases, because all errors here are serious
+      app.exit(1)
```

Comment:
Where can I see the list of errors? I'm worried that there might be non-critical errors.

Reply:
the list is a comment at the start of the function:
```js
this.serverProcess.on('error', err => {
      // The 'error' event is emitted whenever:
      // - The process could not be spawned.
      // - The process could not be killed.
      // - Sending a message to the child process failed.
      // - The child process was aborted via the signal option.
      // ~ https://nodejs.org/api/child_process.html#event-error
```

Reply:
Oops, missed that.

> Sending a message to the child process failed.

I'm not sure about this. Are we unable to proceed when this happens? Have we seen this happen?

Reply:
better exit/crash than undefined behaviour

Reply:
That's for sure. But what I'm getting at is that maybe this happens all the time right now, and Delta Chat keeps working fine, without us knowing it.

Reply:
It will act weird if messages get lost -> like won't react to user action. Better to fail in those cases that we notice the problem and can fix it IMO.

Reply:
I would leave this line out until after 1.52.0. 
But if you're sure it's fine, I won't insist.